qiskit_nature/algorithms/ground_state_solvers/minimum_eigensolver_factories/vqe_ucc_factory.py

```diff
@@ -41,7 +41,7 @@ def __init__(
         self,
         quantum_instance: QuantumInstance,
         optimizer: Optional[Optimizer] = None,
-        initial_point: Optional[Union[np.ndarray, InitialPoint]] = None,
+        initial_point: Optional[Union[np.ndarray, InitialPoint]] = HFInitialPoint(),
```

Comment:
Good point. I actually remember that now as one of the classic Python "gotchas"... I think the original goal of introducing a HFInitialPoint was to make the default arguments more explicit, but I suppose it will still be clear if we explicitly spin up an instance to replace None.